packages/transmute-cli/src/commands/init.js

```diff
@@ -1,12 +1,17 @@
 const run = require('./runner');
 
-export function k8s( dryrun, clusterName ) {
-  let prov_cmd = "$HOME/.transmute/git/transmute/scripts/initializer " + clusterName;
-  console.log( prov_cmd );
-  if (dryrun == 'true' ) {
-    console.log( '<--dry run-->' );
-  }
-  else {
-    run.ner( prov_cmd );
+const path = require('path');
+
+module.exports.k8s = (dryrun, clusterName) => {
+  let prov_cmd = path.join(
```

Comment:
Can we improve this variable name? I assume it has to do with provisioning, but it's a bit vague here.

Reply:
Yeah, this seems like it was copied from the provision.js and should be updated to `init_cmd`

Reply:
not in this PR, I will refactor the rest of this once its been unblocked, as previously stated.